packages/data/src/store/actions.js

```diff
@@ -35,3 +35,21 @@ export function finishResolution( reducerKey, selectorName, args ) {
 		args,
 	};
 }
+
+/**
+ * Returns an action object used in signalling that we should invalidate the resolution cache.
+ *
+ * @param {string} reducerKey   Registered store reducer key.
+ * @param {string} selectorName Name of selector for which resolver should be invalidated.
+ * @param {...*}   args         Arguments to associate for uniqueness.
```

Comment:
Doesn't appear that `args` is a spread argument as documented here. Should be `Array` type?